recipes/refnx/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{%set name = "refnx" %}
+{%set version = "0.0.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/refnx/refnx.git
+  git_rev: 404eceaff24d6b3ae116851dba25beb8069b0092
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
Added. What does the toolchain package do? It's not clear why I have to add this, it's not present in the current recipe example. Could the answer to this be added to an FAQ?